server/plugin.go

```diff
@@ -173,3 +282,11 @@ func (p *Plugin) sendRefreshEvent(userID string) {
 		&model.WebsocketBroadcast{UserId: userID},
 	)
 }
+
+func (p *Plugin) getUserName(userID string) string {
+	userName := "Someone"
+	if user, err := p.API.GetUser(userID); err == nil {
```

Comment:
The convention in Go is to test for `err != nil` -- if you don't, some people will misread this and be confused.